news/2022-07-18-skinning-contest-chromatic-alteration-announcement.md

```diff
@@ -0,0 +1,100 @@
+date: 2022-07-18 12:00:00 +0000
+---
+
+The Skinning Contest is back again for the third time — grab your design tools and join us for the details!
+
+![](/wiki/shared/news/2022-07-18-skinning-contest-chromatic-alteration-announcement/banner.jpg)
+
+You know what they say — third time’s the charm! We are once again inviting all skinning enthusiasts to compete with each other and show off their talent for this iteration of the skinning contest. For those unfamiliar with what this is all about, check out the [wiki index page](/wiki/Contests/Skinning_Contest) for previous iterations.
+
+Straight out of the blizzard that was Tides of Winter, we've decided to forgo a theme for this iteration. However, in an effort to keep the competition up and running, we will be introducing a challenge for you: **Chromatic Alteration** - the details of which can be found down below.
+
+## Challenge: Chromatic Alteration {#challenge}
+
+To make things more interesting, this iteration of the Skinning Contest features a challenge in the form of a limitation:
+
+**The colour palettes of all skin submissions must be limited to the visible spectrum of the following types of colour vision deficiencies**:
+
+- **Protanopia**
+- **Deuteranopia**
+- **Tritanopia**
+
+Bear in mind that your skin must feature **only one** of these colour spectra. Additionally, while shades of grey are contained in these palettes, your skin must not be primarily monochrome.
+
+![](/wiki/shared/news/2022-07-18-skinning-contest-chromatic-alteration-announcement/palettes.png)
+
+Shown above are the spectra and example colour palettes for the three colour vision deficiencies mentioned prior in relation to normal colour vision. These spectra are based on normal colour vision at full saturation and brightness.
+
+## Changes
+
+As per the feedback we've received from you, we've decided to make a couple of adjustments and improvements to the rules:
+
+- **Submissions must not be created as a part of, or involved in a paid commission of any sort**. We encourage contestants to keep their monetary transactions out of the boundaries of the contest in order to ensure a fair and square competitive environment.
+- The cover image appended to your submission must be at least **1280x720 (16:9)**. These will be used on the contest page representing your submission.
+
+## Participation
+
+As always, you are required to create a new skin from scratch in order to participate — there's no point in hosting a competition amongst aged skins after all!
+
+Valid skin submissions must include a link to their forum thread in their [submission post](https://osu.ppy.sh/community/forums/topics/1612259). Said forum thread must be located in the [Work In Progress Skins](https://osu.ppy.sh/community/forums/119) subforum.
+
+Since all entries will have their respective forum threads publicly available, contestants and their submissions will not be anonymized. In fact, we encourage you to show off your work as much as you want!
```

Comment:
since "colour" was used i'm assuming british english

```suggestion
Since all entries will have their respective forum threads publicly available, contestants and their submissions will not be anonymised. In fact, we encourage you to show off your work as much as you want!
```